### DISCUSSION TITLE
[feat] new keybind for navigating the address bar results

### DISCUSSION THREAD

**Discussion by KorigamiK** [zen-browser/desktop#1936] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

I would love to use vim/emacs keys to navigate the results in the address bar 
such as <kbd>Ctrl</kbd>+<kbd>N</kbd>/<kbd>P</kbd> for moving up and down.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

Mozilla Zen 1.0.1-a.3

### What platform are you seeing the problem on?

Linux, macOS - aarch64, macOS - Intel, Windows

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1887/[feat]-new-keybind-for-navigating-the-address-bar-results">ZEN-1887 [feat] new keybind for navigating the address bar results</a></p>

**Reply by mr-cheffy** [reply]
![image](https://github.com/user-attachments/assets/0c2b484e-446e-4c0d-955c-99b064c87b9d)

**Reply by KorigamiK** [reply]
Ah, sorry too much hurry :sob:

**Comment by lmdevv** [comment]
Hello, is there any updates on this? @KorigamiK @mauro-balades 
Thanks!

**Reply by KorigamiK** [reply]
Thanks for reminding, I'll keep this posted as I work on it

**Reply by customtiy13** [reply]
will be excited to see this feat

**Comment by CrynTox** [comment]
Would like to see this feature introduced!

**Comment by ali-aljufairi** [comment]
one of the things stopping me from switching
